src/SelfService/Infrastructure/Api/ApiResourceFactory.cs

```diff
@@ -961,4 +984,85 @@ public TeamListApiResource Convert(List<Team> teams)
+                        action: nameof(InvitationController.GetActiveInvitations),
+                        values: new { userId }
+                    ) ?? "",
+                    rel: "self",
+                    allow: Allow.Get
+                )
+            )
+        );
+    }
+
+    public InvitationListApiResource Convert(IEnumerable<Invitation> invitations, string userId, string targetType)
+    {
+        return new InvitationListApiResource(
+            items: invitations.Select(Convert).ToArray(),
+            links: new InvitationListApiResource.InvitationListLinks(
+                self: new ResourceLink(
+                    href: _linkGenerator.GetUriByAction(
+                        httpContext: HttpContext,
+                        controller: GetNameOf<InvitationController>(),
+                        action: nameof(InvitationController.GetActiveInvitations),
+                        values: new { userId, targetType }
+                    ) ?? "",
+                    rel: "self",
+                    allow: Allow.Get
+                )
+            )
+        );
+    }
+
+    public InvitationApiResource Convert(Invitation invitation)
+    {
+        return new InvitationApiResource(
+            invitation.Id,
+            invitation.Invitee,
+            invitation.Description,
+            invitation.CreatedBy,
+            invitation.CreatedAt.ToUniversalTime().ToString("O"),
+            new InvitationApiResource.InvitationLinks(
+                self: new ResourceLink(
+                    href: _linkGenerator.GetUriByAction(
+                        httpContext: HttpContext,
+                        action: nameof(InvitationController.GetInvitation),
+                        controller: GetNameOf<InvitationController>(),
+                        values: new { id = invitation.Id }
+                    ) ?? "",
+                    rel: "self",
+                    allow: Allow.Get
+                ),
+                accept: new ResourceLink(
+                    href: _linkGenerator.GetUriByAction(
+                        httpContext: HttpContext,
+                        action: nameof(InvitationController.AcceptInvitation),
+                        controller: GetNameOf<TeamController>(),
+                        values: new { id = invitation.Id }
+                    ) ?? "",
+                    rel: "related",
+                    allow: Allow.Post
+                ),
+                decline: new ResourceLink(
+                    href: _linkGenerator.GetUriByAction(
+                        httpContext: HttpContext,
+                        action: nameof(InvitationController.DeclineInvitation),
+                        controller: GetNameOf<TeamController>(),
+                        values: new { id = invitation.Id }
+                    ) ?? "",
+                    rel: "related",
+                    allow: Allow.Post
+                )
```

Comment:
teamcontroller? Is this some future work I have failed to see?

Reply:
More likely a copy-paste issue, don't you think?
I have corrected this.